### PR TITLE
Create the Discord.js topic

### PR DESCRIPTION
**Pull request by gatomo-oficial**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> This topic must be created due to the immense amount of projects built on this module

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/discordjs/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---	
+aliases: discord.js	
+display_name: Discord.js	
+logo: discordjs.png	
+related: discord	
+short_description: discord.js module to interact with the Discord API	
+topic: discordjs	
+---	
+discord.js is a powerful node.js module that allows you to interact with the Discord API very easily. It takes a much more object-oriented approach than most other JS Discord libraries, making your bot's code significantly tidier and easier to comprehend.
```
Comment (gallexi) [inline topics/discordjs/index.md:9]: Overall this topic looks great, thanks for your contribution!!

Would it be possible to reword the description here to be a smidge softer on the other libraries? 😅   This library sounds excellent, but the way this is worded is just a bit more opinionated than we'd prefer to be, I think.

Perhaps
```suggestion
discord.js is a powerful node.js module that allows you to interact with the Discord API very easily. It takes an object-oriented approach, supporting your bot's code to be tidy and easy to comprehend.
```

File: topics/discordjs/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (iancanderson) [inline topics/discordjs/index.md:1]:
```suggestion
---
```

File: topics/discordjs/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---	
+aliases: discord.js	
+display_name: Discord.js	
+logo: discordjs.png	
+related: discord	
+short_description: discord.js module to interact with the Discord API	
+topic: discordjs	
+---
```
Comment (iancanderson) [inline topics/discordjs/index.md:8]:
```suggestion
---
```

**Review by iancanderson — COMMENTED** [review]
I'm not sure, but I think CI might be failing due to the trailing spaces after `---`

**Comment by iancanderson** [comment]
@gatomo-oficial this is getting closer, but there are still issues surfaced in the CI build failure. Feel free to fix those and re-request a review

**Comment by kenyonj** [comment]
Feel free to reopen when ready. Thanks!

**Pull request closed without merging**